sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystConf.scala

```diff
@@ -69,6 +71,7 @@ case class SimpleCatalystConf(
     optimizerMaxIterations: Int = 100,
     optimizerInSetConversionThreshold: Int = 10,
     maxCaseBranchesForCodegen: Int = 20,
+    tableRelationCacheMaxSize: Int = 100,
```

Comment:
this is used for local testing, so 100 might be sufficient? or should we change it to 1000?